l10n_nl_hr_employee_name/view/hr_employee.xml

```diff
@@ -0,0 +1,32 @@
+<openerp>
+  <data>
+    <record id="view_hr_employee_simple_form_firstname" model="ir.ui.view">
+      <field name="name">hr.employee.simplified.form.firstname</field>
+      <field name="model">hr.employee</field>
+      <field
+          name="inherit_id"
+          ref="hr.view_employee_form"
+      />
+      <field name="arch" type="xml">
+          <data>
+              <label for="name" position="replace"/>
+              <xpath expr="//h1[field[@name='name']]" position="replace">
```

Comment:
I think we should do something similar to https://github.com/OCA/l10n-netherlands/pull/128#issue-280988907 here